File: Poin A/Product.java
public class Product{
    
    //Deklarasi Atribut
    private int price;
    private int id_Product;
    
    
    //Constructor
    public Product(int price, int id_Product) {
        this.price = price;
        this.id_Product = id_Product;
    }
    
    
    //Constructor Kosong
    public Product() {
    }

    
    //Get Set Methods
    public int getId_Product() {
        return id_Product;
    }
    public int getPrice() {
        return price;
    }
    public void setPrice(int price) {
        this.price = price;
    }
    public void setId_Product(int id_Product) {
        this.id_Product = id_Product;
    }
    
    //Output Method
    public void output_product(){
        System.out.println("Id Product          : "+ this.getId_Product());
        System.out.println("Price               : "+ this.getPrice());
    }

}

File: Poin B.php/Product.php
<?php

class Product{
	//Deklarasi Atribut
    private $price;
    private $id_product;


	//Constructor
    function __construct(){}

	//Get Set Methods
    function getPrice() {
		return $this->price;
	}

	function setPrice($price){
		$this->price = $price;
	}

    
    function getId_product() {
		return $this->id_product;
	}
	
	function setId_product($id_product){
		$this->id_product = $id_product;
	}


	//Output Method
	function output_product(){
		echo "Id Product			: ". $this->getId_product()."<br/>";
		echo "Price					: ". $this->getPrice()."<br/>";

	}

	//Destructor
    function __destruct()
    {
    }
	

	
}

?>